data/kernels/filmic.cl

```diff
@@ -1052,7 +1052,7 @@ filmic_mask_clipped_pixels(read_only image2d_t in, write_only image2d_t out,
   const float argument = -pix_max * normalize + feathering;
   const float weight = clamp(1.0f / ( 1.0f + native_exp2(argument)), 0.f, 1.f);
 
-  if(4.f > argument) write_imageui(is_clipped, (int2)(0, 0), 1);
+  if(4.f > argument) *is_clipped = 1;
```

Comment:
You take is_clipped as a bool here. As this is the only place setting it to one this should be ok.

Reply:
One more thing, if you have a large part of the image with clipped locations your code might be pretty slow on some machines (AMD iirc had issues)

You might have to use something like one bool per line and collect after reading them all. See the opposed hlr code for an example.

Reply:
OK, I am trying to do a reduction thing.  Bear with me, until now I relied on OpenMP for this.

Reply:
As the existing pr code can't be slower than master and we don't need atomics I am fine with current pr. On Nvidia it won't matter from my experience.

Reply:
Yeah, I just checked and it doesn't seem to matter a lot on Intel either.  A quick benchmarking tells me that this version vs. a proto-reduction run at approximately the same time with extreme parameters.  This should be good to merge then, and we can still improve this if it turns out to be problematic on AMD.